packages/content/templates/nuxt-content.js

```diff
@@ -109,7 +115,8 @@ export default {
       required: true
     }
   },
-  render (h, { data, props }) {
+  render (h, { data, props, parent }) {
+    const { localePath } = parent
```

Comment:
I'm not sure but will it broken if `NuxtContent` component is inside another functional component?